CHANGELOG.md

```diff
@@ -3,6 +3,92 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.11.0
+
+_Aug 3, 2023_
+
+We'd like to offer a big thanks to the 12 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Add new package `@mui/x-tree-view` containing the tree view component
```

Comment:
```suggestion
- ⌚️ Move tree view component from the lab

  The TreeView component have been moved to the MUI X repository.
  They are now accessible in their own packages: `@mui/x-tree-view`.
```

And we would add the link to the blog post when available

Reply:
Should we explain more the lab (maybe a hyperlink) for those who don't know about it? Or simply making `lab` => `@mui/lab`

Reply:
I copied the message from the pickers migration
But I do agree that `lab` => `@mui/lab` is a nice improvement